api/Document.json

```diff
@@ -4515,6 +4765,10 @@
             "safari_ios": {
               "version_added": null
             },
+            "samsunginternet_android": {
+              "alternative_name": "webkitIsFullScreen",
```

Comment:
Another weird issue in the original data...  Filing a PR for this for Chrome, but if `version_added` is `false`, it's impossible for it to have an alternate name.

```suggestion
```